Scripts/UI/MainMenuActivator.cs

```diff
@@ -68,47 +80,72 @@ public bool activatorButtonMoveAway
 		Coroutine m_ActivatorMoveCoroutine;
 		Vector3 m_AlternateActivatorLocalPosition;
 
+		bool m_Disabled;
+		Material m_IconMaterial;
+		Color m_EnabledColor;
+
 		public Transform rayOrigin { private get; set; }
 		public Transform menuOrigin { private get; set; }
 		public Node? node { get; set; }
 
-		public event Action<Transform> hoverStarted;
-		public event Action<Transform> hoverEnded;
 		public event Action<Transform, Transform> selected;
 
-		public void OnPointerEnter(PointerEventData eventData)
+		public bool disabled
 		{
-			// A child may have used the event, but still reflect that is was hovered
-			var rayEventData = eventData as RayEventData;
-			if (rayEventData != null && hoverStarted != null)
-				hoverStarted(rayEventData.rayOrigin);
+			get { return m_Disabled; }
+			set
+			{
+				if (value != m_Disabled)
+				{
+					m_Icon.sharedMaterial.color = value ? m_DisabledColor : m_EnabledColor;
 
-			if (eventData.used)
-				return;
+					if (value)
+						SetHighlight(false);
+				}
 
-			if (m_HighlightCoroutine != null)
-				StopCoroutine(m_HighlightCoroutine);
+				m_Disabled = value;
+			}
+		}
 
-			m_HighlightCoroutine = null;
-			m_HighlightCoroutine = StartCoroutine(Highlight());
+		public string tooltipText
+		{
+			get { return m_Disabled ? "Main Menu Hidden" : null; }
+		}
+
+		public Transform tooltipTarget { get { return m_TooltipTarget; } }
+		public Transform tooltipSource { get { return m_TooltipSource; } }
+		public TextAlignment tooltipAlignment { get { return TextAlignment.Center; } }
+
+		void Awake()
+		{
+			m_IconMaterial = MaterialUtils.GetMaterialClone(m_Icon);
+			m_EnabledColor = m_IconMaterial.color;
+		}
+
+		public void OnPointerEnter(PointerEventData eventData)
+		{
+			if (eventData.used || m_Disabled)
+				return;
+
+			SetHighlight(true);
 			this.Pulse(node, m_HoverPulse);
 		}
 
 		public void OnPointerExit(PointerEventData eventData)
 		{
-			// A child may have used the event, but still reflect that is was hovered
-			var rayEventData = eventData as RayEventData;
-			if (rayEventData != null && hoverEnded != null)
-				hoverEnded(rayEventData.rayOrigin);
-
 			if (eventData.used)
 				return;
 
+			SetHighlight(false);
+		}
+
+		void SetHighlight(bool highlighted)
+		{
 			if (m_HighlightCoroutine != null)
 				StopCoroutine(m_HighlightCoroutine);
 
 			m_HighlightCoroutine = null;
```

Comment:
Not sure. @dunity wrote it. I can get rid of it, but I was just moving this code around.